packages/react-hooks/src/useScreen/state.ts

```diff
@@ -1,6 +1,6 @@
 import { breakpoints } from "@fremtind/jkl-core";
 
-interface State {
+export interface State {
```

Comment:
Kanskje gjøre det litt tydeligere hva det er, siden den eksporteres? Innser at det må endres litt flere steder i så fall, men føles nesten alltid verdt det å være eksplisitt 😊

```suggestion
export interface ScreenState {
```

Reply:
Kan jo bare 
`import { State as ScreenState } from "somewhere-over-the-rainbow"`

Tipper man normalt bare trenger å importere en State der det skal brukes, så tror egentlig ikke det er et problem.